src/mixins/itext.svg_export.js

```diff
@@ -131,7 +131,7 @@
           timeToRender = this._hasStyleChanged(actualStyle, nextStyle);
         }
         if (timeToRender) {
-          style = this._getStyleDeclaration(lineIndex, i) || { };
+          style = this.getStyleDeclaration(lineIndex, i) || { };
```

Comment:
we cannot rename this as part of the PR. _getStyleDeclaration will stay named like this for now.